api/database/connection.js

```diff
@@ -0,0 +1,52 @@
+const { Sequelize } = require("sequelize");
+
+require("dotenv").config();
+
+const createDbConnection = (
+  databaseName,
+  userName,
+  databasePassword,
+  dbHost,
+  dbDialect
+) => {
+  return new Sequelize(databaseName, userName, databasePassword, {
+    host: dbHost,
+    dialect: dbDialect,
+  });
+};
+
+const checkConnectionToDb = async (sequelize) => {
+  try {
+    await sequelize.authenticate();
+    console.log("Connected to PostgresSQL database");
+  } catch (error) {
+    console.error("Connection to PostgresSQL database failed", error);
+    throw error;
+  }
+};
+
+const disconnectFromDb = async (sequelize) => {
+  try {
+    await process.on("SIGINT", () => sequelize.close());
```

Comment:
why did u close the connection like that and how is it different from closing it the simple way?